pr_agent/servers/github_action_runner.py

```diff
@@ -58,6 +58,13 @@ async def run_action():
         print("OPENAI_KEY not set")
     if OPENAI_ORG:
         get_settings().set("OPENAI.ORG", OPENAI_ORG)
+
+    if get_settings().get("aws.AWS_ACCESS_KEY_ID"):
+        assert get_settings().aws.AWS_SECRET_ACCESS_KEY and get_settings().aws.AWS_REGION_NAME, "AWS credentials are incomplete"
+        os.environ["AWS_ACCESS_KEY_ID"] = get_settings().aws.AWS_ACCESS_KEY_ID
+        os.environ["AWS_SECRET_ACCESS_KEY"] = get_settings().aws.AWS_SECRET_ACCESS_KEY
+        os.environ["AWS_REGION_NAME"] = get_settings().aws.AWS_REGION_NAME
```

Comment:
Why do we need to inject `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_REGION_NAME` to environment variables? @trathailoi

Reply:
@trathailoi 

@KennyDizi is correct. with github action, just give the values as direct env variables.

Reply:
I've tried many times, many ways, but didn't work.

Reply:
aws is [supported](https://github.com/Codium-ai/pr-agent/issues/1050#issuecomment-2244241459) in github action, so this PR will not be merged.

you can try pr-agent pro if you prefer a hosted version of PR-Agent, provided by CodiumAI.

all the best

Reply:
it always tries to look for OPENAI_KEY, I don't set it though.
<img width="580" alt="Screenshot 2024-07-26 at 22 37 12" src="https://github.com/user-attachments/assets/9647eef5-6bc3-4d7a-a1f2-ef8639c7c601">
<img width="1065" alt="Screenshot 2024-07-26 at 22 38 26" src="https://github.com/user-attachments/assets/089ede97-c54f-4e4b-ad07-b7203aeebd12">

Reply:
where does the error message say anything about openAI ?

Reply:
the part saying `read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.'`. I meant to use aws bedrock, but the error was telling me to check out the OpenAI docs.

Reply:
I'm so sorry, I meant to fork and create the PR on that fork to test the approach myself, but I accidentally create this one. I'm going to closed this PR.

Reply:
i cannot debug you personal deployment.
somewhere you are giving an openai key.
maybe in your secrets, or in a configuration file.

debug from CLI.

In any case, this PR is plain wrong

Reply:
you are right, I was setting openai key in my secrets. I removed it, and the 429 error code was gone.
but now it is 401 instead 😂
<img width="1143" alt="Screenshot 2024-07-26 at 23 36 43" src="https://github.com/user-attachments/assets/f91dccbe-e788-49fd-bb14-650323ad1657">

Reply:
I can confirm that the openai key was not set. But looks like the env variables that I set was not working.
<img width="1204" alt="Screenshot 2024-07-26 at 23 30 13" src="https://github.com/user-attachments/assets/672dbc56-4480-448b-acd9-b9759c4adb78">

here is my setting
<img width="636" alt="Screenshot 2024-07-26 at 23 29 29" src="https://github.com/user-attachments/assets/8dbd9440-6939-4ee6-a4b4-1e8497a489d5">
<img width="804" alt="Screenshot 2024-07-26 at 23 29 03" src="https://github.com/user-attachments/assets/e85130ec-e45c-45ef-8717-9e522e4cf9b2">

I don't put any configuration file in the repo btw.

Reply:
could you please give me some advice on any next things that I should do to investigate the issue ? 😢